master/values.yaml

```diff
@@ -0,0 +1,113 @@
+andi:
+  enabled: true
+  redis:
+    host: master-redis-master
+  strimzi:
+    kafka:
+      brokers: streaming-service-kafka-bootstrap:9092
+carpenter:
+  enabled: true
+  redis:
+    host: master-redis-master
+discovery-api:
+  enabled: true
+  replicaCount: 1
+  redis:
+    host: master-redis-master
+  ldap:
+    host: ldap
+discovery-streams:
+  enabled: false
+discovery-ui:
+  enabled: true
+flair:
+  enabled: true
+  redis:
+    host: master-redis-master
+  strimzi:
+    kafka:
+      brokers: streaming-service-kafka-bootstrap:9092
+forklift:
+  enabled: true
+  redis:
+    host: master-redis-master 
+  strimzi:
+    kafka:
+      brokers: streaming-service-kafka-bootstrap:9092
+kdp:
+  enabled: false
+genesis:
+  enabled: true
+  redis:
+    host: master-redis-master 
+  kafka:
+    public_brokers: streaming-service-kafka-bootstrap:9092
+odo:
+  enabled: true
+  redis:
+    host: master-redis-master
+reaper:
+  enabled: true
+  redis:
+    host: master-redis-master
+  strimzi:
+    kafka:
+      brokers: streaming-service-kafka-bootstrap:9092
+  vault:
+    endpoint: ""
+redis:
+  enabled: true
+  usePassword: false
+  cluster:
+    enabled: false
+streisand:
+  enabled: true
+  redis:
+    host: master-redis-master
+  kafka:
+    pipeline_brokers: streaming-service-kafka-bootstrap:9092
+    public_brokers: streaming-service-kafka-bootstrap:9092
+valkyrie:
+  enabled: true
+  replicaCount: 1
+  redis:
+    host: master-redis-master 
+voltron:
+  enabled: true
+  replicaCount: 1
+  redis:
+    host: master-redis-master
+  strimzi:
+    kafka:
+      brokers: streaming-service-kafka-bootstrap:9092
+kafka:
+  broker: streaming-service-kafka-bootstrap:9092
+  strimziCluster: streaming-service
+  storageSize: 5Gi
+  defaultPartitions: 1
+  defaultReplicas: 1
+  topics:
```

Comment:
We shouldn't be using:
```
- dataset-registry
- streaming-raw
- streaming-validated
- streaming-transformed
- organization-raw
```

And I'm not sure about `smoke-test`. Do we still need that topic?

Reply:
I deleted the extra topics